css-positioning/absolute.css

```diff
@@ -0,0 +1,50 @@
+.meme {
+  width: 300px;
+  height: 300px;
+  background-image: url("images/one-does-not-simply.jpg");
+  background-size: cover;
+
+  /* your code here */
+  position: relative;
+}
+
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  right: 0;
+  right: 15px;
```

Comment:
You should never declare the same property twice in the same rule. Keep just one of these, and delete the other.